checks/readme.php

```diff
@@ -76,13 +77,31 @@ function check_for_default_text() {
 	}
 
 	function check_for_warnings() {
-		if ( ! empty( $this->readme->warnings ) ) {
+		$warnings = $this->readme->warnings ?? [];
+		$warning_keys = array_keys( $warnings );
+		$ignored_warnings = [
+			'contributor_ignored'
+		];
+
+		/**
+		 * Filter the list of ignored readme parser warnings.
+		 *
+		 * @since 0.2.2
+		 *
+		 * @param array  $ignored_warnings Array of ignored warning keys.
+		 * @param Parser $readme The readme object.
+		 */
+		$ignored_warnings = apply_filters( 'plugin_check_readme_warnings_ignored', $ignored_warnings, $this->readme );
```

Comment:
Just for good measure we should probably type cast this as an array, incase users return a single string.
```suggestion
		$ignored_warnings = (array) apply_filters( 'plugin_check_readme_warnings_ignored', $ignored_warnings, $this->readme );
```